src/test/scala/com/fortysevendeg/android/sqlite/statement/StatementArgumentsSpec.scala

```diff
@@ -0,0 +1,288 @@
+package com.fortysevendeg.android.sqlite.statement
+
+import java.sql.SQLException
+
+import com.fortysevendeg.android.sqlite._
+import org.specs2.mock.Mockito
+import org.specs2.mutable.Specification
+import org.specs2.specification.Scope
+
+trait StatementArgumentsSpecification
+  extends Specification
+  with Mockito {
+
+  trait StatementArgumentsScope
+    extends Scope {
+
+  }
```

Comment:
As Codacy said, you can just define the trait as:

`trait StatementArgumentsScope extends Scope`